packages/engine/README.md

```diff
@@ -74,35 +74,15 @@ Depending on your needs, different dependencies are required. Building this proj
 
 ### Required dependencies
 
-- The Rust Compiler
-
-  - We recommend installing and using rustup, following the [instructions on the Rust-Lang website](https://www.rust-lang.org/tools/install)
-  - hEngine runs on the Nightly toolchain. The version is managed by the [rust-toolchain.toml](./rust-toolchain.toml) file. To verify, run `rustup show` from the [engine](.) directory.
 
 - CMake [3.X.X >= 3.21.2]
 
   - CMake installation guidance from the [CMake page](https://cmake.org/install/) or if on macOS through [brew](https://brew.sh/)
 
-- a C++ compiler, pkg-config, openssl development files (see [Possible Dependencies and Debugging](#possible-dependencies-and-debugging))
-
```

Comment:
This needs to stay

Reply:
Thanks. Can we still keep the line break?

Reply:
Added back in https://github.com/hashintel/hash/pull/414/commits/52c36f1540c563fc1c9196741c6918fabc13cc4b